src/lib.rs

```diff
@@ -31,3 +32,47 @@ pub mod xds;
 
 pub mod hyper_util;
 pub mod test_helpers;
+
+#[cfg(feature = "gperftools")]
+extern crate gperftools;
+
+use extensions::Extension;
+use tracing::info;
+
+// #[global_allocator]
+// static GLOBAL: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
+
+// #[global_allocator]
+// static GLOBAL: tcmalloc::TCMalloc = tcmalloc::TCMalloc;
+
+pub fn entry(extension: Option<Box<dyn Extension + Sync + Send>>) -> anyhow::Result<()> {
+    telemetry::setup_logging();
+    let config: config::Config = config::parse_config(extension)?;
```

Comment:
I think it should be placed after the match below

Reply:
this part of was moved as is from main.rs